internal/pkg/ecs/ecs_test.go

```diff
@@ -328,6 +328,84 @@ func TestClient_DescribeService(t *testing.T) {
 	}
 }
 
+func TestClient_ForceUpdateService(t *testing.T) {
+	const (
+		mockApp     = "mockApp"
+		mockEnv     = "mockEnv"
+		mockSvc     = "mockSvc"
+		mockSvcARN  = "arn:aws:ecs:us-west-2:1234567890:service/mockCluster/mockService"
+		mockCluster = "mockCluster"
+		mockService = "mockService"
+	)
+	getRgInput := map[string]string{
+		deploy.AppTagKey:     mockApp,
+		deploy.EnvTagKey:     mockEnv,
+		deploy.ServiceTagKey: mockSvc,
+	}
+
+	tests := map[string]struct {
+		setupMocks func(mocks clientMocks)
+
+		wantedError error
+	}{
+		"return error if failed to get service tasks": {
```

Comment:
nit: maybe this better describes the test case -
```suggestion
		"return error if failed to update service": {
```

Reply:
Forgot to change the description 🤦